include/libopencm3/stm32/g0/lptimer.h

```diff
@@ -39,6 +39,43 @@
 #define LPTIM2				LPTIM2_BASE
 /**@}*/
 
+/** LPTIM_CFGR2 LPTIM configuration register 2 */
+#define LPTIM_CFGR2(tim_base)			MMIO32((tim_base) + 0x24)
+
+/** @addtogroup lptim_cr
+@{*/
+/** COUNTRST Counter reset **/
+#define LPTIM_CR_COUNTRST		(1 << 3)
+/** RSTARE Reset after read enable **/
+#define LPTIM_CR_RSTARE		(1 << 4)
+/**@}*/
+
+/** @defgroup lptim_cfgr2 LPTIM_CFGR2 Configuration Register 2
+@{*/
+
+#define LPTIM_CFGR2_IN2SEL_SHIFT	4
+#define LPTIM_CFGR2_IN2SEL_MASK		0x03
+/** @defgroup lptim_cfgr2_in2sel IN2SEL Input 2 selection
+ @{*/
+#define LPTIM_CFGR2_IN2SEL_MUX0 		0
+#define LPTIM_CFGR2_IN2SEL_MUX1 		1
+#define LPTIM_CFGR2_IN2SEL_MUX2 		2
+#define LPTIM_CFGR2_IN2SEL_MUX3 		3
+/**@}*/
+
+#define LPTIM_CFGR2_IN1SEL_SHIFT	0
+#define LPTIM_CFGR2_IN1SEL_MASK		0x03
+/** @defgroup lptim_cfgr2_in1sel IN1SEL Input 1 selection
+@{*/
+#define LPTIM_CFGR2_IN1SEL_MUX0 		0
+#define LPTIM_CFGR2_IN1SEL_MUX1 		1
+#define LPTIM_CFGR2_IN1SEL_MUX2 		2
+#define LPTIM_CFGR2_IN1SEL_MUX3 		3
```

Comment:
I'm honestly not really convinced that defines for 0,1,2,3, that are .... 0,1,2,3 are actually useful.

Reply:
i'll drop that.